### PR TITLE
Babel config + prettier

### PR DESCRIPTION
**Pull request by fbarbare**

### CODE REVIEW

File: next/babel.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const { PUBLIC_URL, NEXT_BUILD_ID } = process.env;
+
+process.env.ASSET_PREFIX = '/_next';
+
+if (PUBLIC_URL && NEXT_BUILD_ID) {
+  process.env.ASSET_PREFIX = `${PUBLIC_URL}/_next/${NEXT_BUILD_ID}`;
+}
+
+const envVars = Object.keys(process.env)
+  .filter(key => key.indexOf('NEXT_APP_') === 0)
```
Comment (rameshanandakrishnan) [inline next/babel.js]: Using `!== -1` would be safer or why not use `key.contains('NEXT_APP_')`

File: next/config.js [hunk 2]
```diff
@@ -2,5 +2,11 @@ const path = require('path');
 const withCSS = require('@zeit/next-css');
 
 module.exports = withCSS({
-  distDir: path.join('..', '..', 'build')
+  distDir: path.join('..', '..', 'build'),
+  assetPrefix: process.env.PUBLIC_URL || '',
+  generateBuildId: async () => {
```
Comment (rameshanandakrishnan) [inline next/config.js]: why is this async?

File: next/babel.js (hunk 1 above)
Comment (ricardo-devis-agullo) [reply next/babel.js]: I think you mean `includes` :P

But in this case, the alternative would be `key.startsWith('NEXT_APP_')`

File: next/config.js (hunk 2 above)
Comment (fbarbare) [reply next/config.js]: Because that's how they wrote it in they docs: https://github.com/zeit/next.js/#configuring-the-build-id :)

**Pull request merged**